src/daft-plan/src/physical_plan.rs

```diff
@@ -183,6 +184,18 @@ impl PhysicalPlan {
                     .call1((upstream_iter, aggs_as_pyexprs, groupbys_as_pyexprs))?;
                 Ok(py_iter.into())
             }
+            PhysicalPlan::Coalesce(Coalesce {
+                input,
+                num_from,
+                num_to,
+            }) => {
+                let upstream_iter = input.to_partition_tasks(py, psets)?;
+                let py_iter = py
+                    .import(pyo3::intern!(py, "daft.execution.physical_plan"))?
+                    .getattr(pyo3::intern!(py, "coalesce"))?
+                    .call1((upstream_iter, *num_from as i64, *num_to as i64))?;
```

Comment:
Nit: I _think_ this `usize` -> `i64` cast should be done automatically by pyo3, at least we've been relying on that behavior elsewhere. Any advantage to doing it explicitly here?
```suggestion
                    .call1((upstream_iter, *num_from, *num_to))?;
```